templates/simple/simple.sh

```diff
@@ -243,11 +249,17 @@ ${JSB_TMP}/ifdh.sh cp -D {{fname}} ${CONDOR_DIR_INPUT}
     ${JSB_TMP}/ifdh.sh cp {{tfname}} {{tflocal}}
     tar --directory .unwind_{{loop.index0}} -xzvf {{tflocal}}
     {%if loop.first%}
+      INPUT_TAR_DIR_LOCAL=`pwd`/.unwind_{{loop.index0}}
+      export INPUT_TAR_DIR_LOCAL
       INPUT_TAR_FILE=`pwd`/.unwind_{{loop.index0}}
```

Comment:
Looking over this again, isn't this line inconsistent with the docs (I know you didn't change it now, but I just noticed this)?  

Both #223 and the helptext say that `$INPUT_TAR_FILE` points to the actual tarfile in the PNFS case, not the directory.  So for the CVMFS case, both `$INPUT_TAR_FILE` and `$INPUT_TAR_DIR_LOCAL` point to the directory, since there is no tarfile to point at, but for the PNFS case, `$INPUT_TAR_FILE` points at the actual tarball (so `{{tflocal}}`, it appears), and `$INPUT_TAR_DIR_LOCAL` points to the directory containing the unpacked tarball.

Am I understanding that correctly?  If so, do you think it's a reasonable way to do it, or do you think the PNFS and CVMFS cases should not differ?  Basically, it's a tossup between honoring the meaning of a variable named `$INPUT_TAR_FILE`, and having that point to an actual tarfile if possible, or keeping the behavior consistent no matter what the dropbox method. 

What are your thoughts?